### PR TITLE
First step of Add NFT

### PR DESCRIPTION
**Pull request by ihor-stack**
First step of Add NFT
https://freedcamp.com/view/2956183/tasks/panel/task/40029527
image

### CODE REVIEW

**Review by ngvtuan — COMMENTED** [review]
I think this PR has 2 important things that need to do: 
- Merge the `new_development` branch to this PR
- Use `tailwind CSS` like other branches.


UI issue 1
1. Missing Close icon
2. `3.` `4.` Doesn't same with the design
5. remove/hide the left side because it doesn't has in the new design

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "test": "jest",
     "format": "eslint --ignore-path=.gitignore --ext .ts,.tsx --fix . && prettier --ignore-path=.gitignore --loglevel=error --write .",
     "check-eslint": "eslint --ignore-path=.gitignore --ext .ts,.tsx --max-warnings=0 .",
-    "check-prettier": "prettier --ignore-path=.gitignore --list-different .",
+    "check-prettier": "prettier --ignore-path=.gitignore --list-different --write .",
```
[the remark this answers is not in the document]
Comment (br4in3x) [reply package.json:28]: @ihor-stack please don't change anything related to scripts or CI, it breaks builds for other devs

File: src/features/artReg/components/ArtRegForm.module.css [hunk 2]
```diff
@@ -1,3 +1,48 @@
+@font-face {
```
Comment (br4in3x) [inline src/features/artReg/components/ArtRegForm.module.css:1]: I think for setup should be in the [index.css](https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/index.css)

File: src/features/artReg/components/ArtRegForm.tsx [hunk 3]
```diff
@@ -19,11 +19,10 @@ export function ArtRegForm(): JSX.Element {
...
     <ScrollPane offsetHeight={0}>
-      <div className={cx(cstyles.xlarge, cstyles.padall, cstyles.center)}>
-        Art Registration
-      </div>
-      <div className={styles.container}>
-        <div className={cstyles.well}>{stepMap[step]}</div>
+      <div className={styles.artRegFormContainer}>
+        <div className={styles.artRegFormMainBlock}>
+          <div className={styles.artRegFormStep}>{stepMap[step]}</div>
+        </div>
```
Comment (br4in3x) [inline src/features/artReg/components/ArtRegForm.tsx:25]: We switched to Styled Components. Please follow the example in any of these components: https://github.com/pastelnetwork/pastel-electron-wallet/tree/master/src/common/components

File: src/features/expertConsole/TerminalConsole.tsx [hunk 4]
```diff
@@ -242,36 +242,34 @@ function TerminalConsole(props: TConsoleProps): JSX.Element {
     }
```
Comment (br4in3x) [inline src/features/expertConsole/TerminalConsole.tsx:242]: Why is this file being changed?

File: src/features/expertConsole/rpc-services/utils.ts [hunk 5]
```diff
@@ -169,9 +169,7 @@ export const METHODS = [
```
Comment (br4in3x) [inline src/features/expertConsole/rpc-services/utils.ts:169]: Why is this file being changed?

File: src/features/pastelPaperWalletGenerator/PastelPaperWalletModalSlice.ts [hunk 6]
```diff
@@ -39,7 +39,5 @@ export const pastelPaperWalletModalSlice = createSlice({
```
Comment (br4in3x) [inline src/features/pastelPaperWalletGenerator/PastelPaperWalletModalSlice.ts:39]: Unrelated changed. Please revert.

File: src/features/pastelPhotopea/PastelPhotopeaModalSlice.tsx [hunk 7]
```diff
@@ -23,7 +23,5 @@ export const pastelPhotopeaModalSlice = createSlice({
```
Comment (br4in3x) [inline src/features/pastelPhotopea/PastelPhotopeaModalSlice.tsx:23]: Unrelated changed. Please revert.

File: src/features/receive/AddressBlock.tsx [hunk 8]
```diff
@@ -56,9 +56,8 @@ export const AddressBlock = (props: IAddressBlockProps): JSX.Element => {
     [],
```
Comment (br4in3x) [inline src/features/receive/AddressBlock.tsx:56]: Unrelated changed. Please revert.

File: src/features/squooshTool/squooshToolModalSlice.ts [hunk 9]
```diff
@@ -23,7 +23,5 @@ export const squooshToolModalSlice = createSlice({
```
Comment (br4in3x) [inline src/features/squooshTool/squooshToolModalSlice.ts:23]: Unrelated changed. Please revert.

File: src/legacy/rpc.ts [hunk 10]
```diff
@@ -342,11 +342,9 @@ export default class RPC {
```
Comment (br4in3x) [inline src/legacy/rpc.ts:342]: Unrelated changed. Please revert.

**Review by br4in3x — CHANGES_REQUESTED** [review]

**Pull request closed without merging**